pkg/smb/nodeserver_test.go

```diff
@@ -34,6 +34,19 @@ import (
 	"k8s.io/utils/mount"
 )
 
+func getSMBAccessDeniedErrorOnWindows(file string) error {
+	return status.Error(codes.Internal,
+		fmt.Sprintf("volume(vol_1##) mount \"test_source\" on %#v"+
+			" failed with smb mapping failed with error: rpc error: code = "+
+			"Unknown desc = NewSmbGlobalMapping failed. output: \"New-SmbGlobalMapping"+
+			" : Access is denied. \\r\\nAt line:1 char:190\\r\\n+ ... ser, "+
+			"$PWord;New-SmbGlobalMapping -RemotePath $Env:smbremotepath -Cred ...\\r\\n+"+
+			"                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"+
+			"\\r\\n    + CategoryInfo          : PermissionDenied: (MSFT_SmbGlobalMapping:ROOT/Microsoft/...mbGlobalMapping)"+
+			" [New-SmbGlobalMapping], Cim \\r\\n   Exception\\r\\n    + FullyQualifiedErrorId : "+
+			"Windows System Error 5,New-SmbGlobalMapping\\r\\n \\r\\n\", err: exit status 1", file))
+}
```

Comment:
@andyzhangx These error messages seem pretty flaky. I'm getting a slightly different message on my local Windows machine, but Github Action gives a different one. Any ideas about how we can go about asserting these errors?

Reply:
use error msg matching should be enough, e.g. match `Access is denied`

Reply:
We're not getting `Access is Denied` error on Github Actions. Instead we're getting `The parameter is incorrect`

```
Unexpected error: rpc error: code = Internal desc = volume(vol_1##) mount "test_source" on "D:\\a\\csi-driver-smb\\csi-driver-smb\\pkg\\smb\\source_test" failed with smb mapping failed with error: rpc error: code = Unknown desc = NewSmbGlobalMapping failed. output: "New-SmbGlobalMapping : The parameter is incorrect. \r\nAt line:1 char:190\r\n+ ... ser, $PWord;New-SmbGlobalMapping -RemotePath $Env:smbremotepath -Cred ...\r\n+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : InvalidArgument: (MSFT_SmbGlobalMapping:ROOT/Microsoft/...mbGlobalMapping) [New-SmbGloba \r\n   lMapping], CimException\r\n    + FullyQualifiedErrorId : Windows System Error 87,New-SmbGlobalMapping\r\n \r\n", err: exit status 1         
```

Reply:
it failed here: https://github.com/kubernetes-csi/csi-proxy/blob/25491d0543e53381aab69ee5e5fa114182787b43/internal/os/smb/api.go#L68
could be different error description on Windows, you can try above command to verify.